examples/spatialdata_demo.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
<p>Rather than passing an enviorjnment, please just add installation instructions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/napari-spatialdata/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>